runtime/parachains/src/inclusion.rs

```diff
@@ -398,7 +398,7 @@ impl<T: Config> Pallet<T> {
 		signed_bitfields: UncheckedSignedAvailabilityBitfields,
 		disputed_bitfield: DisputedBitfield,
 		core_lookup: impl Fn(CoreIndex) -> Option<ParaId>,
-	) -> Result<Vec<(CoreIndex, CandidateHash)>, DispatchError> {
+	) -> Vec<(CoreIndex, CandidateHash)> {
```

Comment:
We should consider returning `Result<Vec<(CoreIndex)>, Vec<(CoreIndex)>>` to be able to easily evaluate if the sanitization caused any changes and be able to warn outside this fn call and also keep the `assert_noop!` storage check macro.

Reply:
We can use `assert_storage_noop!` if we need to check for storage root changes https://crates.parity.io/frame_support/macro.assert_storage_noop.html

What storage changes though do you want the system to keep track of? It seems like the calling logic doesn't have any handling based on if storage changes are made here

Reply:
I can't follow. The `Ok()/Err()` would be independent of the `storage_noop` and we should verify that no storage was written in case we do error (which from the tests is never the case), which I missed. So please discard that part :]